cmd/cca/verify_as_relyingparty.go

```diff
@@ -91,6 +95,15 @@ previous invocation to "evcli cca create" command.
 		"api-server", "s", "", "URL of the Veraison verification API",
 	)
 
+	relyingPartyIsInsecure = cmd.Flags().BoolP(
```

Comment:
it might be helpful, if the README.md can be elevated to express what the default behaviour is and also availability & how to use the supplied arguments

Reply:
I've aded a note to README-CCA.md